build/media-suite_helper.sh

```diff
@@ -2224,76 +2220,47 @@ create_cmake_toolchain() {
         printf '%s\n' "${toolchain_file[@]}" > "$LOCALDESTDIR"/etc/toolchain.cmake
 }
 
-get_signature() {
-    # get_signature 96865171 0F3BE490
-    # adds keys to gpg keychain for verifying
-    for keyserver in keys.openpgp.org pool.sks-keyservers.net keyserver.ubuntu.com pgp.mit.edu; do
-        gpg --keyserver "$keyserver" --receive-keys "$@" && break
-    done > /dev/null 2>&1
-}
-
-check_signature() {
-    # check_signature -k 96865171 gnutls-3.6.8.tar.xz.sig gnutls-3.6.8.tar.xz
-    # check_signature -k 1528635D8053A57F77D1E08630A59377A7763BE6 http://libsdl.org/release/SDL2-2.0.10.tar.gz.sig SDL2-2.0.10.tar.gz
-    # run in the same directory as the files. Works with .sig and some .asc
-    # file needs to start with -----BEGIN PGP SIGNATURE-----
-    local key=()
-    while true; do
-        case $1 in
-        -k) key+=("$2") && shift 2 ;; # keys to retrieve using get_signature
-        --)
-            shift
-            break
-            ;;
-        *) break ;;
-        esac
-    done
-    local sigFile=$1
-    shift
-
-    # Get name of sig file
-    local sigFileName=${sigFile##*/}
-    sigFileName=${sigFileName:-"$(/usr/bin/curl -sI "$sigFile" | grep -Eo 'filename=.*$' | sed 's/filename=//')"}
-    [[ -z $sigFileName ]] && echo "Sig file not set" && return 1
-
-    # Download sig file if url/cp file if local file
-    if ! do_wget -c -r -q "$sigFile" "$sigFileName" && [[ -f $sigFile ]]; then
-        sigFile="$(
-            cd_safe "$(dirname "$sigFile")"
-            printf '%s' "$(pwd -P)" '/' "$(basename -- "$sigFile")"
-        )"
-        [[ ${sigFile%/*} != "$PWD" ]] && cp -f "$sigFile" "$sigFileName" > /dev/null 2>&1
-    fi
-
-    # Retrive keys
-    [[ -n ${key[0]} ]] && get_signature "${key[@]}"
-
-    # Verify file is correct
-    # $? 1 Bad sig or file integrity compromised
-    # $? 2 no-pub-key or no file
-    gpg --auto-key-retrieve --keyserver hkps://hkps.pool.sks-keyservers.net --verify "$sigFileName" "$@" > /dev/null 2>&1
-    case $? in
-    1) do_exit_prompt "Failed to verify integrity of ${sigFileName%%.sig}" ;;
-    2) do_exit_prompt "Failed to find gpg key or no file found for ${sigFileName%%.sig}" ;;
-    esac
-}
+do_fix_pkgconfig_abspaths() {
+    # in case the root was moved, this fixes windows abspaths
+    mkdir -p "$LOCALDESTDIR/lib/pkgconfig"
+    # pkgconfig keys to find the wrong abspaths from
+    local _keys="(prefix|exec_prefix|libdir|includedir)"
+    # current abspath root
+    local _root
+    _root=$(cygpath -m "$LOCALDESTDIR")
+    # find .pc files with Windows abspaths
+    grep -ElZR "${_keys}=[^/$].*" "$LOCALDESTDIR"/lib/pkgconfig | \
+        # find those with a different abspath than the current
+        xargs -0r grep -LZ "$_root" | \
+        # replace with current abspath
+        xargs -0r sed -ri "s;${_keys}=.*$LOCALDESTDIR;\1=$_root;g"
+    unset _keys _root
```

Comment:
Vaguely sure this line is not needed and has no effect overall since these are declared as local to the function and should be automatically cleaned up when bash exits the function